gui/wxpython/lmgr/giface.py

```diff
@@ -204,6 +204,9 @@ def __init__(self, lmgr):
         # Signal emitted to request updating of map
         self.updateMap = Signal('LayerManagerGrassInterface.updateMap')
 
+        # Signal emitted when workspace is changed
+        self.workspaceChanged = Signal('LayerManagerGrassInterface.workspaceChanged')
+
```

Comment:
So this needs to be added also to StandaloneGrassInterface

Reply:
Sorry, of course, you are right.